pyproject.toml

```diff
@@ -3,7 +3,7 @@ requires = ["flit_core ~=3.2"]
 build-backend = "flit_core.buildapi"
 
 [project]
-name = "universal-transfer-operator"
+name = "apache-airflow-provider-transfers"
 dynamic = ["version"]
 description = """Universal Transfer Operator transfers all the data that could be read from the source Dataset into
```

Comment:
The description talks about Universal Transfer Operator while the project name is apache-airflow-provider-transfers. Should we mentioned how they are related in the description?

Reply:
changed the wording